.github/workflows/docker.yaml

```diff
@@ -0,0 +1,93 @@
+name: Docker 
+
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+
+env:
+  DOCKER_HUB_SLUG: driveone/onedrive
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        flavor: [ fedora, debian, alpine ]
+        include:
+          - flavor: fedora
+            dockerfile: ./contrib/docker/Dockerfile
+            platforms: linux/amd64
+          - flavor: debian
+            dockerfile: ./contrib/docker/Dockerfile-debian
+            platforms: linux/amd64,linux/arm64,linux/arm/v7
+          - flavor: alpine
+            dockerfile: ./contrib/docker/Dockerfile-alpine
+            platforms: linux/amd64,linux/arm64
+
+    steps:
+      - name: Check out code from GitHub
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+
+      - name: Docker meta
+        id: docker_meta
+        uses: marcelcoding/ghaction-docker-meta@v1
+        with:
+          tag-edge: true
+          images: |
+            ${{ env.DOCKER_HUB_SLUG }}
+            ghcr.io/${{ github.repository }}
+          tag-semver: |
+            {{version}}
+            {{major}}.{{minor}}
+          flavor: ${{ matrix.flavor }}
+          main-flavor: ${{ matrix.flavor == 'debian' }}
+
+      - uses: docker/setup-qemu-action@v1
+        if: matrix.platforms != 'linux/amd64'
+
+      - uses: docker/setup-buildx-action@v1
+
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ matrix.flavor }}-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-${{ matrix.flavor }}
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        if: github.event_name != 'pull_request'
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        if: github.event_name != 'pull_request'
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
+
+      - name: Build
+        uses: docker/build-push-action@v2
+        with:
+          context: ./onedrive

```

Comment:
This line should be a ".". I tested the file, sorry for the inconvenience. I am only on my phone and did not saw that this is a pr. :)